### DISCUSSION TITLE
Set release in native native code, overriden by JS

### DISCUSSION THREAD

**Discussion by makirby** [getsentry/sentry-react-native#2916] [Q&A]
Hi there,

We moved to initialising and setting the release string on the native side.

Unfortunately when we initialse Sentry on the JS side (with `autoInitializeNativeSdk: false`), all subsequent events  recieve the default release string, even though we set our release string on the native side.

Is there a way to opt-out of this and force the SDK to use the release string we set on the native side?

Cheers

**Comment by krystofwoldrich** [comment]
Hi,
thank you for the message,

the SDK gets the release information from the native side if there is no release information set in javascript.

Here is the logic: https://github.com/getsentry/sentry-react-native/blob/960186fa5e9b541ffa471aa839601c3bafca2b26/src/js/integrations/release.ts#L50

What do you mean by `default release string`?
